src/Attachment/AttachmentManager.php

```
@@ -269,14 +275,18 @@ public static function getList($issue_id)
                     iat_iss_id=? AND
                     iat_usr_id=usr_id AND
                     iat_min_role <= ?';
+        if ($not_id) {
+            $stmt .= ' AND
+                    iat_not_id = ?';
+            $params[] = $not_id;
+        }
         $stmt .= '
                  ORDER BY
                     iat_created_date ASC';
-        $params = [$issue_id, User::getRoleByUser($usr_id, $prj_id)];
         try {
             $res = DB_Helper::getInstance()->getAll($stmt, $params);
         } catch (DatabaseException $e) {
-            return '';
+            return [];
         }
```

remove the try-catch. lets stop hiding errors.